mmdet/models/dense_heads/paa_head.py

```diff
@@ -516,76 +516,92 @@ def _get_targets_single(self,
             label_channels=1,
             unmap_outputs=True)
 
-    def _get_bboxes_single(self,
-                           cls_scores,
-                           bbox_preds,
-                           iou_preds,
-                           mlvl_anchors,
-                           img_shape,
-                           scale_factor,
-                           cfg,
-                           rescale=False,
-                           with_nms=True):
+    def _get_bboxes(self,
```

Comment:
1 comments the length of the list
2 (num_points * num_classes, H, W) in docstr maybe wrong?